src/track.rs

```diff
@@ -379,3 +399,108 @@ impl Tracks {
         self.client.get_stream(&uri.join("?"), num_pages)
     }
 }
+
+/// Provides access to operations available for a track's comments
+pub struct Comments {
```

Comment:
I think we should either use different naming for the comment types or represent it as an enum.
```rust
enum Comments {
  User { client, user_id },
  Track { client, track_id }
}
```

You could share the `iter` and `get` implementation and just have specific `path` implementations then.

Reply:
Yes, I like this much better.